lms/djangoapps/shoppingcart/models.py

```diff
@@ -1928,7 +1928,9 @@ def purchased_callback(self):
 
     def additional_instruction_text(self):
         verification_reminder = ""
+        refund_reminder_msg = _("You have up to two weeks into the course to unenroll and receive a full refund. ")
         is_enrollment_mode_verified = self.course_enrollment.is_verified_enrollment()  # pylint: disable=E1101
+        is_professional_mode_verified = self.course_enrollment.is_professional_enrollment()  # pylint: disable=E1101
```

Comment:
please use the text name to disable `disable=no-member` and if you could fix up the one above as well that would be super.